### ISSUE TITLE
Small warning when compiling with emscripten

### ISSUE THREAD

**Issue by raysan5** [jkuhlmann/cgltf#74]
Just a small detail, when compiling with emscripten I got a warning:
```
cgltf.h:446:3: warning: redefinition of typedef 'cgltf_node' is a C11 feature [-Wtypedef-redefinition]
```
`typedef` can be just avoided in the struct definition because it has already been defined previously.

I can send a PR if you want.

**Comment by jkuhlmann** [comment]
PR would of course be appreciated. :-) Otherwise I’ll look into it next week.

**Comment by Calinou** [comment]
This issue can be closed now that #75 has been merged :slightly_smiling_face:
